build/pipe.go

```diff
@@ -1,3 +1,19 @@
+// Copyright (c) 2019 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository at
+//     https://github.com/direct-state-transfer/dst-go/NOTICE
```

Comment:
- [ ] This line should read
  ```go
  // https://github.com/direct-state-transfer/dst-go
  ```
- [ ] As you didn't update the NOTICE file during your first contribution, please verify that your contributions are covered or update it as part of this pull request